src/main/java/ch/njol/skript/doc/JSONGenerator.java

```diff
@@ -372,6 +392,18 @@ private static JsonArray cleanPatterns(String... strings) {
 		return convertToJsonArray(strings);
 	}
 
+	private void cacheEvents() {
+		for (SkriptEventInfo<?> eventInfo : Skript.getEvents()) {
+			for (Class<? extends Event> event : eventInfo.events) {
+				if (events.containsKey(event)) {
```

Comment:
```suggestion
				if (!events.containsKey(event)) {
```
oopsie, this should fix it not generating. after fixing this it looked good to me